src/StaticFAQ/FAQExtraInfo/FAQExtraInfoButton.js

```diff
@@ -71,6 +79,7 @@ const FAQExtraInfoButton = ({ category, children, history }: Props) => {
                 <button
                   className={`extraInfoRadioButton ${isActive ? 'active' : ''}`}
                   onClick={() => {
+                    category === 'baggage' ? trackBaggage : trackBoardingPass;
```

Comment:
fine, but now the function wouldn't be executed

Reply:
🤦‍♂️